fendermint/vm/topdown/src/lib.rs

```diff
@@ -31,6 +31,8 @@ pub type BlockHash = Bytes;
 
 /// The null round error message
 pub(crate) const NULL_ROUND_ERR_MSG: &str = "requested epoch was a null round";
+/// Default topdown proposal height interval
+pub(crate) const DEFAULT_PROPOSAL_INTERVAL: BlockHeight = 10;
```

Comment:
This would mean that we catch up with the root chain in bursts of 5 minute jumps. Why should it work like that?